src/chrome/content/rules/Red_Hat.xml

```diff
@@ -126,12 +126,9 @@
 	<target host="webmail.redhat.com" />
 	<target host="www.redhat.com" />
 
-	<!--	Complications:
-				-->
+	<!--	Complications:      -->
 	<target host="www.cz.redhat.com" />
 	<target host="et.redhat.com" />
-	<target host="investors.redhat.com" />
```

Comment:
The target (as well as the jobs.redhat.com one) is just moved to [here](https://github.com/EFForg/https-everywhere/pull/4688/files#diff-612abbdc45c9a25f4bf6477a15bf5e76R129), since it no longer adds complications.